From ddd34dc8096b8be3998bc9c6fcd53852d9773a3e Mon Sep 17 00:00:00 2001
From: homoluden <asm@metaphysics-it.net>
Date: Wed, 23 Oct 2013 08:54:09 +0400
Subject: [PATCH 2/3] Main window size increased

---
 FukamiDemo/MainWindow.xaml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/FukamiDemo/MainWindow.xaml b/FukamiDemo/MainWindow.xaml
index 224ced3..4dc5b7c 100644
--- a/FukamiDemo/MainWindow.xaml
+++ b/FukamiDemo/MainWindow.xaml
@@ -2,7 +2,7 @@
         xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
         xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
         xmlns:rdr="clr-namespace:Renderers;assembly=Renderers"
-        Title="Fukami Demo" Height="350" Width="525">
+        Title="Fukami Demo" Height="600" Width="800">
     <Grid>
         <Grid.RowDefinitions>
             <RowDefinition />
-- 
1.8.4.msysgit.0

